apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/PairAssertionSamples.kt

```diff
@@ -0,0 +1,53 @@
+package ch.tutteli.atrium.api.fluent.en_GB.samples
+
+import ch.tutteli.atrium.api.fluent.en_GB.first
+import ch.tutteli.atrium.api.fluent.en_GB.second
+import ch.tutteli.atrium.api.fluent.en_GB.toBe
+import ch.tutteli.atrium.api.verbs.internal.expect
+import kotlin.test.Test
+
+class PairAssertionSamples {
+
+    // subject under test
+    private val sut = 1 to "one"
+
+    @Test
+    fun firstToBe() {
+
+        expect(sut).first.toBe(1)
+
+        fails {
+            expect(sut).first.toBe(2)
+        }
+    }
+
+    @Test
+    fun firstToBeAsFunction() {
+
+        expect(sut).first { toBe(1) }
+
+        fails {
+            expect(sut).first { toBe(2) }
```

Comment:
Please document and show that a subsequent assertion is still about the pair. 
Same same in the other example where the subsequent assertiom is about the first. Maybe use isGreaterThan and subsequent isLessThan

Reply:
`Please document and show that a subsequent assertion is still about the pair.` didn't get it, why we need it?

`Same same in the other example where the subsequent assertiom is about the first. Maybe use isGreaterThan and subsequent isLessThan` ok

Reply:
added parameter hints, used isGreaterThan and isLessThan

Reply:
Now on my PC, I'll rewrite one of your samples so that you see what I mean. 
Basically, we have two syntax:
- first
- first {} 

and we would like to show to the users what the difference is also in the sample

Reply:
ok thanks